table/server/master/src/main/java/alluxio/master/table/AlluxioCatalog.java

```diff
@@ -118,17 +118,8 @@ public boolean attachDatabase(JournalContext journalContext, String udbType,
               .setDbName(dbName)
               .putAllConfig(map).build()).build());
 
-      try {
-        mDBs.get(dbName).sync(journalContext);
-      } catch (Exception e) {
```

Comment:
Why remove this processing? If we fail to sync the db, we should not have attached the db, right?